Source/RapyutaSimulationPlugins/Public/Tools/SimulationState.h

```diff
@@ -56,32 +58,33 @@ class RAPYUTASIMULATIONPLUGINS_API ASimulationState : public AActor
 public:
     /**
      * @brief Construct a new ASimulationState object
-     * 
+     *
      */
     ASimulationState();
 
 public:
     /**
      * @brief Start ROSservices.
-     * 
-     * @param InROS2Node @sa [ROS2Node](https://rclue.readthedocs.io/en/devel/doxygen_generated/html/d6/dcb/class_a_r_o_s2_node.html)
+     *
+     * @param InROS2Node @sa

```

Comment:
can you  avoid change line after @sa or change line before @sa

Reply:
Yeah, unclear why somehow clang-format does it.